### PR TITLE
add examples of how to get user info from req.auth

### PR DESCRIPTION
**Pull request by victoriaxyz**
> [!IMPORTANT]
> 🔎 Previews:
>
> - https://clerk.com/docs/pr/1648

### This PR:

- show how to get user info from `req.auth`
- need to update types for Express: https://clerkinc.slack.com/archives/C05GH6N5LLS/p1729878244842629

### CODE REVIEW

**Review by royanger — APPROVED** [review]
🚀 Looks good

**Review by alexisintech — APPROVED** [review]

**Comment by royanger** [comment]
> @alexisintech @royanger
> 
> I updated the Express example (i.e., removed `any` type and `return`): [89c9223](https://github.com/clerk/clerk-docs/commit/89c92238e00c9dba4613055573c834f8f293bd8e)
> 
> The `@types/express` 5.0.0v doesn't allow `return res.json()`. See [here](https://github.com/expressjs/express/issues/5987#issuecomment-2428333462).

To be clear, you can do `res.json()` if you want it in the code. If just needs to be one of these two options, at least from I've seen. I haven't tested yet.

```ts
return void res.status(400).json({ message: error });
```

```ts
res.status(400).json({ message: error });
return
```

Did you want to include a `res.json()` in the example? If you're having trouble getting this working we can Tuple.

File: docs/references/express/overview.mdx [hunk 1]
```diff
@@ -130,9 +130,17 @@ import express from 'express'
 const app = express()
 const port = 3000
 
-app.get('/users', requireAuth(), async (req, res) => {
-  const users = await clerkClient.users.getUserList()
-  return res.json(users)
+app.get('/users', async (req, res) => {
+  // Get the userId from the Auth object
```
Comment (alexisintech) [inline docs/references/express/overview.mdx:134]:
```suggestion
  // Get the `userId` from the `Auth` object
```

**Review by royanger — APPROVED** [review]
Looks good to me

**Pull request merged**